src/jit/importer.cpp

```diff
@@ -1740,47 +1741,57 @@ GenTreePtr          Compiler::impRuntimeLookupToTree(CORINFO_RUNTIME_LOOKUP_KIND
         ctxTree = gtNewLclvNode(info.compTypeCtxtArg, TYP_I_IMPL); // Exact method descriptor as passed in as last arg
     }
 
+#ifdef FEATURE_READYTORUN_COMPILER
```

Comment:
I'd prefer that we remove these `#ifdef`s and simply rely on `opts.IsReadyToRun`.